student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super
```

Comment:
- [ ] When creating a student, kindly add the `age`, `name`, `parent_permission = true` in the parameters. 
- [ ] Also, when calling super, kindly add the parameters `age`, `name`, `parent_permission` to the super method.